content/en/apps/tutorials/multi-facility-users.md

```diff
@@ -0,0 +1,201 @@
+
+<br clear="all">
+
+{{< figure src="new-chw-area/new-chw-area.png" link="new-chw-area/new-chw-area.png" class="right col-6 col-lg-8" >}}
+
+Select the **Health Facility** on the left-hand side. Click on the **Blue Action Button** and select **New Area**.
+
+<br clear="all">
+
+{{< figure src="new-chw-area/create-new-person.png" link="new-chw-area/create-new-person.png" class="right col-6 col-lg-8" >}}
+
+Select the option that lets you create a new person within the form. This person will automatically become the primary contact for the created place.
+
+When filling in the required fields select **CHW** as the role. 
+
+Once you have filled the required fields, click **Next** to go to the next section.
+
+<br clear="all">
+
+{{< figure src="new-chw-area/name-after-primary-contact.png" link="new-chw-area/name-after-primary-contact.png" class="right col-6 col-lg-8" >}}
+
+You will get an option to name the Place after the created contact person or name it yourself. If you select **Yes**, the new place will be named `<contact-name>'s Area`. For example `Jane Doe's Area`.
+
+<br clear="all">
+
+{{< figure src="new-chw-area/created-chw-area.png" link="new-chw-area/created-chw-area.png" class="right col-6 col-lg-8" >}}
+
+Once you submit, a new CHW Area will be created. On the right-hand you should see the CHW Area name, the primary contact of the CHW Area, and the Health Facility that the CHW Area belongs to.
+
+<br clear="all">
+
+*****
+
+#### 3. Create Second Health Facility. 
+
+<br clear="all">
+
+{{< figure src="new-facility/skip-primary-contact.png" link="new-facility/skip-primary-contact.png" class="right col-6 col-lg-8" >}}
+
+For the second Health Facility, we will skip creating or assigning a primary contact and focus on creating the second Health Facility.
+
+<br clear="all">
+
+{{< figure src="new-facility/enter-second-facility-name.png" link="new-facility/enter-second-facility-name.png" class="right col-6 col-lg-8" >}}
+
+Enter the details of the Health Facility and submit the form.
+
+<br clear="all">
+
+{{< figure src="new-facility/multiple-facilities.png" link="new-facility/multiple-facilities.png" class="right col-6 col-lg-8" >}}
+
+You should see the two Health Facilities appear on the left-hand side.
+
+<br clear="all">
+
+### 4. Create the CHW Supervisor User
+
+Let's create a CHW Supervisor user who's linked to the CHW Supervisor contact we created earlier.
+
+<br clear="all">
+
+{{< figure src="new-chw-supervisor/app-settings.png" link="new-chw-supervisor/app-settings.png" class="right col-6 col-lg-8" >}}
+
+Go to the **hamburger menu** and select **App Settings**.
+
+<br clear="all">
+
+{{< figure src="new-chw-supervisor/add-user.png" link="new-chw-supervisor/add-user.png" class="right col-6 col-lg-8" >}}
+
+When you are on the **App Settings** page, select **Users** on the left-hand side and then select **Add User** on the right-hand side.
```

Comment:
```suggestion
When you are on the **App Management** page, select **Users** on the left-hand side and then select **Add User** on the right-hand side.
```
In `default` config and [cht docs](https://docs.communityhealthtoolkit.org/apps/features/admin/), it's called different: 


<img width="793" alt="Screenshot 2024-07-29 at 3 18 12 PM" src="https://github.com/user-attachments/assets/7fb2bcce-f52e-4c8c-9920-22c39cfa7e54">